grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -24,12 +24,15 @@ import com.twcable.grabbit.proto.NodeProtos.Properties
 import com.twcable.grabbit.proto.NodeProtos.Property
 import groovy.transform.CompileStatic
```

Comment:
Interactions are commonly defined in a `setup:` block. There are some edge-case rules, but generally you declare them wherever it makes it easiest to read.

https://spockframework.github.io/spock/docs/1.0/interaction_based_testing.html#_where_to_declare_interactions